packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -495,17 +495,13 @@ impl CreateTransaction {
         self.tx.salt()
     }
 
-    pub fn bytecode_witness_index(&self) -> u8 {
+    pub fn bytecode_witness_index(&self) -> u16 {
         *self.tx.bytecode_witness_index()
     }
 
     pub fn storage_slots(&self) -> &Vec<StorageSlot> {
         self.tx.storage_slots()
     }
-
-    pub fn bytecode_length(&self) -> u64 {
```

Comment:
We should add a `BREAKING CHANGE` section and list all of the breaking changes there.